simgui-ds.json

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
Is this file needed? Seems like it was generated by running the simulator. Maybe we could add this (and the other one) to the `.gitignore`.